src/VisualStudio/LiveShare/Impl/Client/RemoteLanguageServiceWorkspace.cs

```diff
@@ -218,13 +256,13 @@ public Document GetOrAddDocument(string filePath)
                 return null;
             }
 
-            var folderName = Path.GetFileNameWithoutExtension(_remoteRootPath);
+            var folderName = Path.GetFileNameWithoutExtension(remoteWorkspaceRoot);
```

Comment:
What's the significance of making different projects for different roots?

Reply:
Generally existing files are associated with the project already and never hit this path.  New files do hit this (as in files created during the liveshare session), so it decides (rather arbitrarily) that they belong to a project with the name of the first remote root path.  This changes it so it's at least in the correct remote root path.

But project info on the client is generally not used, so it doesn't matter much (and will matter even less once I fix the other navbar bugs).